config/config.go

```diff
@@ -329,12 +331,37 @@ func (c *config) PingTests() bool {
 func (c *config) AllowedIPs() []string {
 	mu.RLock()
 	defer mu.RUnlock()
-	return c.AllowIPs
+	return ips(c.AllowIPs)
 }
 
 // AllowedIPs returns a reloadable list of block-listed ips to avoid running speed tests for
 func (c *config) BlockedIPs() []string {
 	mu.RLock()
 	defer mu.RUnlock()
-	return c.BlockIPs
+	return ips(c.BlockIPs)
+}
+
+func ips(ips []string) []string {
+	hosts := []string{}
+	for _, ip := range ips {
+		if ipv4Addr, ipv4Net, err := net.ParseCIDR(ip); err != nil {
```

Comment:
will you name this just `ipAddr` or `ipv46Address` or something that makes it clear it handles ipv6 too?